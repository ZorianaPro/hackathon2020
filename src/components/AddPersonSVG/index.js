import React from 'react';

const AddPersonSVG = (props) => (
  <svg className="AddPersonSVG" {...props}>
    <use xlinkHref="#AddPersonSymbol" />
  </svg>
);

export default AddPersonSVG;
