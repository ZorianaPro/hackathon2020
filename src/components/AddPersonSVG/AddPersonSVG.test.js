import React from "react";
import { shallow } from "enzyme";
import AddPersonSVG from "./";

describe("AddPersonSVG", () => {
  it("renders without crashing", () => {
    expect(shallow.bind(shallow, <AddPersonSVG />)).not.toThrow();
  });

  it("renders as expected", () => {
    expect(shallow(<AddPersonSVG />)).toMatchSnapshot();
  });
});
