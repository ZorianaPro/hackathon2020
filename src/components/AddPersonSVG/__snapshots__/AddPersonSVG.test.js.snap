// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPersonSVG renders as expected 1`] = `
<svg
  className="AddPersonSVG"
>
  <use
    xlinkHref="#AddPersonSymbol"
  />
</svg>
`;
